src/arkode/arkode_ls.c

```diff
@@ -2167,13 +2199,15 @@ int arkLsDQJtimes(N_Vector v, N_Vector Jv, realtype t,
                             &ark_mem, &arkls_mem);
   if (retval != ARK_SUCCESS)  return(retval);
 
+  SUNAssignSUNCTX(ark_mem->sunctx);
+
   /* Initialize perturbation to 1/||v|| */
-  sig = ONE/N_VWrmsNorm(v, ark_mem->ewt);
+  sig = ONE/SUNCheckCallLastErrNoRet(N_VWrmsNorm(v, ark_mem->ewt));
```

Comment:
Given that `SUNCheckCallLastErrNoRet` expands into a multi-line macro, this line looks dangerous.  Are you confident that this routine is well-tested?

Reply:
I believe this expands to
```
sig = ONE/N_VWrmsNorm(v, ark_mem->ewt); SUNCheckCallNoRet(SUNGetLastErr(SUNCTX));
``` 
however
```
sig = ONE/SUNCheckCallLastErrNoRet(N_VWrmsNorm(v, ark_mem->ewt));
```
is a awkward to read because it's not clear where the result of `N_VWrmsNorm` ends up (without looking at the macro). I image this was done since it's easy to do a find and replace on functions without a return value whereas changing the original line to the macro expansion would be more involved but more readable.